packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -255,6 +282,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
           bounds={ bounds }
           directionalHintFixed={ directionalHintFixed }>
           <div style={ contextMenuStyle } ref={ (host: HTMLDivElement) => this._host = host } id={ id } className={ css('ms-ContextualMenu-container', className) }>
+            { title ? <div className={ classNames.title } role='heading' aria-level={ 1 }> { title } </div> : (undefined) }
```

Comment:
Could this just be `{ title && <div className={ classNames.title } role='heading' aria-level={ 1 }> { title } </div> }`

Reply:
I think this one still needs addressing @Boris-Em ?

Reply:
Good call. Thanks!